<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="formGridLayout">
      <item row="10" column="1">
       <widget class="QSpinBox" name="sbWindowSize">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Левая граница</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDateTimeEdit" name="dtMinDt">
        <property name="displayFormat">
         <string>dd.MM.yy HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pbSelectFiles">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Размер окна</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Папка экспорта</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Выберите файлы для обработки</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Правая граница</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="4">
       <widget class="QLineEdit" name="leOutputFolder">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="11" column="3">
       <widget class="QSpinBox" name="sbMarmettFilter">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="12" column="5">
       <widget class="QPushButton" name="pbFolderExport">
        <property name="text">
         <string>Открыть</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QDateTimeEdit" name="dtMaxDt">
        <property name="displayFormat">
         <string>dd.MM.yy HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QSpinBox" name="sbResampleFrequency"/>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="sbMedianFilter">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Фильтр marmett</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLineEdit" name="leInputFolder">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Фильтр median</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="6">
       <layout class="QGridLayout" name="glGraph"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Частота ресемплирования</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Размер сдвига</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="5">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Параметры кумулятивного спектра</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QSpinBox" name="sbOverlapSize">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="value">
         <number>856</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="6">
       <widget class="QTextEdit" name="teFilesList">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Временной интервал обработки</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QPushButton" name="pbCalculation">
        <property name="text">
         <string>Обработка</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Параметры визуализации</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Минимальная частота</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QDoubleSpinBox" name="deMinFrequency">
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="15" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Максимальная частота</string>
        </property>
       </widget>
      </item>
      <item row="15" column="3">
       <widget class="QDoubleSpinBox" name="deMaxFrequency">
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="15" column="4">
       <widget class="QPushButton" name="pbSpectrumRefresh">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
